pages/tools/braavos.mdx

```diff
@@ -0,0 +1,49 @@
+import { Steps } from 'nextra-theme-docs'
+
+# Braavos Overview
+[Braavos](https://www.braavos.app) is a StarkNet-powered self-custodial wallet prioritizing security and affordability, it incorporates account abstraction, multi-factor authentication, and hardware signing. Moreover, Braavos allows you to managee tokens, NFTs, and dApp interactions across chains.
```

Comment:
managee -> manage

You can download the Grammarly extension on vs code, it's pretty good.